.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI
+
+on:
+  push:
```

Comment:
Just noticed. I don't see value in running the CI check on push. Updates to an open PR would be handled by `pull_request` event target, so the added benefit of `push` is checking non-PR pushes. What's the reason to do that?

Reply:
Based on your comment, there is no any :) I will remove it.